.vib/valkey-cluster/ginkgo/go.mod

```diff
@@ -0,0 +1,57 @@
+module test-redis-chart
```

Comment:
```suggestion
module test-valkey-cluster-chart
```

Reply:
I update this to `test-valkey-chart` the same way as `redis` and `redis-cluster` that share `test-redis-chart`